android-app/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher"
         android:supportsRtl="true"
-        android:usesCleartextTraffic="true"
+        android:usesCleartextTraffic="false"
```

Comment:
I dont think this works. Actually, we need clear traffic to be enabled when we are developing, because we use `http` instead of `https`. I think we need to do this change as soon as we are building the production app. In that case we should also change the android:debuggable to `false`. Anyways, I think its too early for us to change this value, as we are still in development. In production it will work without clear traffic, because we will have an `https` protocol